ml-agents-envs/mlagents_envs/env_utils.py

```diff
@@ -106,6 +106,9 @@ def launch_executable(file_name: str, args: List[str]) -> subprocess.Popen:
         logger.debug(f"Running with args {args}")
         # Launch Unity environment
         subprocess_args = [launch_string] + args
+        # std_out_option = DEVNULL means the outputs will not be displayed on terminal.
+        # std_out_option = None is default behavior: the outputs are displayed on terminal.
```

Comment:
Is this backwards? `DEVNULL` is the default behavior right?

Reply:
No, if no option is passed, then it will default to None = print to the console.